modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/AbstractPageMemoryMvPartitionStorage.java

```diff
@@ -684,14 +684,10 @@ private Cursor<BinaryRow> internalScan(Predicate<BinaryRow> keyFilter, UUID txId
     @Override
     public @Nullable RowId closestRowId(RowId lowerBound) throws StorageException {
         try {
-            Cursor<VersionChain> cursor = versionChainTree.find(new VersionChainKey(lowerBound), null);
-
-            if (!cursor.hasNext()) {
-                return null;
+            try (Cursor<VersionChain> cursor = versionChainTree.find(new VersionChainKey(lowerBound), null)) {
```

Comment:
Do you need a nested `try` here? Can it be merged with the outer `try`?

Reply:
I'll check